pong-game/backend/app/game_service/consumers.py

```diff
@@ -241,8 +244,10 @@ async def join_lobby(self, data):
 		self.current_group = f"lobby_{room_name}"
 		if player_alias in active_lobbies[room_name]["players"]:
 			await self.send(json.dumps({
-				"type": "error",
-				"message": "player is already in lobby"
+				"type": "rejoin",
+				"message": "player is already in lobby",
+				"player1": f"{active_lobbies[room_name]['players'][0]}",
+				"player2": f"{active_lobbies[room_name]['players'][1]}"
 				}))
 			return
```

Comment:
You likely explained this to me already this week, so apologies in advance, but what is the point of this change?

FYI, I started working on the players disconnecting/rejoining games on the consumer and gameRoom level, so if you had ideas regarding this or already started implementing some parts, feel free to reach out.

Reply:
Hmm i have no idea regarding the change lol, i mean i don't know why we need it specifically as it's Alex who asked for it, 
I'm assuming it's in the front end, When you are trying to join a lobby you are already in, as it could happen if you disconnect from the lobby, front end would need player information to set the display right, rather than just a you are in lobby won't make it abble to display for exemple opponent data, this is just me assuming stuff tho.